ui/src/org/pentaho/di/ui/trans/steps/ivwloader/IngresVectorwiseLoaderDialog.java

```diff
@@ -391,7 +390,7 @@ protected TextVar addStandardTextVar( String labelMessageKey, Control prevContro
     int vertPad = verticalPadding;
     Label label = addStandardLabel( labelMessageKey, prevControl );
     verticalPadding = vertPad;
-    TextVar targetControl = new TextVar( transMeta, shell, SWT.SINGLE | SWT.LEFT | SWT.BORDER );
+    TextVar targetControl = new PasswordTextVar( transMeta, shell, SWT.SINGLE | SWT.LEFT | SWT.BORDER );
```

Comment:
Don't agree with this change. addStandardTextVar is called to add many text fields, not just password text fields. This would make all text fields in the VW loader password fields. This needs to be reverted.